cmake/emu/qemu.cmake

```diff
@@ -70,7 +70,7 @@ endif()
 list(APPEND QEMU_FLAGS -serial chardev:con)
 
 # Connect semihosting console to the console chardev if configured.
-if(CONFIG_SEMIHOST_CONSOLE)
+if(CONFIG_ARM_SEMIHOST)
```

Comment:
is this change needed?

Reply:
Yes, `-semihosting-config enable=on` is needed for semihosting to work